c_src/membrane_ffmpeg_swresample_plugin/converter.h

```diff
@@ -1,8 +1,12 @@
 #pragma once
 
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wall"
+#pragma GCC diagnostic ignored "-Wextra"
```

Comment:
I'd restrict using these pragmas only for excluding third party headers from warning

Reply:
These pragmas can be removed entirely